site/content/includes/quickstart/node-3/node_exporter-3.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./node_exporter --collector.textfile.directory=/node-3/ --web.listen-address 127.0.0.1:8081
```

Comment:
Should this be `8082`?

Reply:
It's actually, pretty arbitrary, but I made the port number match the node 8081, 2, 3 etc